src/v2/Components/NavBar/Menus/MoreNavMenu.tsx

```diff
@@ -2,10 +2,15 @@ import { Menu, MenuItem } from "v2/Components/Menu"
 import { AnalyticsSchema } from "v2/Artsy"
 import { useTracking } from "v2/Artsy/Analytics/useTracking"
 import React from "react"
+import { userHasLabFeature } from "v2/Utils/user"
+import { useSystemContext } from "@artsy/reaction/dist/Artsy"
```

Comment:
I think vscode might have picked the wrong import, should be "v2/Artsy" no?

Reply:
oof good catch thank u